wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSessionBuilder.swift

```diff
@@ -222,7 +223,7 @@ struct ZMUserSessionBuilder {
 
     // MARK: UserSesssionDependencies
 
-    private func buildUserSessionDependencies(coreDataStack: CoreDataStack) -> UserSessionDependencies {
+    private func buildUserSessionDependencies(coreDataStack: CoreDataStack, analyticsSessionConfiguration: AnalyticsSessionConfiguration?) -> UserSessionDependencies {
```

Comment:
Is this still needed?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/e12bacb446dc277c68c1c3efba1ebb121183ea68